local_main_simulated.py

```diff
@@ -0,0 +1,15 @@
+from hovor.core import run_interaction, simulate_interaction
+from local_run_utils import *
+
+
+def run_local_conversation(output_files_path):
```

Comment:
Does this function execute any differently than `simulate_local_conversation`?

Reply:
it calls `initialize_local_run_simulated` instead of `initialize_local_run`